auditlog/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,18 @@
+Rules
+-----
+
+Go to `Settings / Technical / Audit / Rules` to subscribe rules. A rule defines
+which operations to log for a given data model.
+
+.. image:: static/description/rule.png
```

Comment:
Hi @max3903 : you should use ``../`` syntax. (relative path). It will working in all cases. (on github, on apps store, on your odoo instance, etc...)
basically : 
`` .. image:: static/description/rule.png``
by 
`` .. image:: ../static/description/rule.png``

Reply:
for more information : https://github.com/OCA/maintainer-tools/pull/397

Reply:
Done